src/Resolvers/Microsoft.NET.Sdk.WorkloadManifestReader/Microsoft.NET.Sdk.WorkloadManifestReader.csproj

```diff
@@ -25,8 +22,7 @@
   </ItemGroup>
 
   <ItemGroup>
-    <PackageReference Include="System.Text.Json" Condition="'$(UseSystemTextJson)'=='True'" />
-    <PackageReference Include="Newtonsoft.Json" Condition="'$(UseSystemTextJson)'!='True'" />
+    <PackageReference Include="System.Text.Json" />
```

Comment:
Move this into the .NET Framework section as System.Text.Json is already inbox on modern .NET. I'm still surprised that this doesn't have a version override on it as other projects in this PR have that.